### ISSUE TITLE
Empty Json has been created.

### ISSUE THREAD

**Issue by santhosh12992** [stevetarver/excel-as-json#7]
Hi Steve,
I have tried converting excel to json... and added this code in my config file(protractor).
var convertExcel = require('excel-as-json').processFile;
convertExcel("D:\\Protractor\\Grunt_Pro\\data.xlsx", "D:\\Protractor\\Grunt_Pro\\test\\sample1.json",false,function(err, data) {
        if(err) console.log(err);
    });

my data.xlsx contains three rows with header value1 and value2. the next two rows has values as similar to shown below.
|value1|value2|
|1        |         2|
|3        |         4|

However json file also created(sample1.json) but it contains only two braces "[]". If needed i will share you more details. Kindly help me to sort out this.

**Comment by stevetarver** [comment]
These test cases are in convertFileSpec.coffee. From your description, it sounds like the excel package is not returning data. What does a log statement about line 149 of excel-as-json.coffee print? If nothing, it is a problem in the excel package. Otherwise, please provide your test project and I will see if I can reproduce the problem.

**Comment by jeril10** [comment]
Hi Steve, 
I am also facing the same issue

```
convertExcel(path.join(__dirname + '/../public/uploads/excel_file.xlsx'), undefined, false, function(err, data) {
    if(err) {
    	console.log("JSON conversion failure ");
    }
    console.log(data);
`});`
```
The returned data is '[]'

**Comment by steventarver** [comment]
As mentioned above, these are basic test cases covered in the convertFileSpec.coffee. This implies the usual suspects are file not found or using an xlsx format that the excel dependency cannot read. You can insert a log message to see these or provide your project and I will look at it in spare time.

**Comment by stevetarver** [comment]
Closing this issue as neither commenter has provided a test project and these simple cases are very similar to existing test cases that pass.

**Comment by stevetarver** [comment]
Of course, just after I close this issue, I am able to reproduce the problem. I have updated the 'excel' dependency and released version 1.0.4. 

If you are still interested, please try the new version and verify it resolves your problems.

**Comment by SophiaHan6** [comment]
Hi I have run into this issue.

It worked for me about one month ago, but empty json is created for me now

Please help!

@steventarver
